kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/convert/BigQuerySchemaConverterTest.java

```diff
@@ -609,6 +609,30 @@ public void testDescription() {
     assertEquals(bigQueryExpectedSchema, bigQueryTestSchema);
   }
 
+    @Test
+    public void testDescriptionInField() {
+        final String fieldName = "WithDoc";
+        final String fieldDoc = "test documentation";
+
+        com.google.cloud.bigquery.Schema bigQueryExpectedSchema =
+                com.google.cloud.bigquery.Schema.of(
+                        com.google.cloud.bigquery.Field.newBuilder(fieldName,
+                                LegacySQLTypeName.STRING)
+                                .setMode(com.google.cloud.bigquery.Field.Mode.REQUIRED)
+                                .setDescription(fieldDoc)
+                                .build()
+                );
+
+        Schema kafkaConnectTestSchema =
+                SchemaBuilder.struct()
+                        .field(fieldName, SchemaBuilder.string().parameter("io.confluent.connect.avro.field.doc." + fieldName,fieldDoc).build())
+                        .build();
+
+        com.google.cloud.bigquery.Schema bigQueryTestSchema =
+                new BigQuerySchemaConverter(false).convertSchema(kafkaConnectTestSchema);
+        assertEquals(bigQueryExpectedSchema, bigQueryTestSchema);
+    }
```

Comment:
Should we add another case that ensures backwards compatibility by converting a Connect schema that has both a standard doc and an Avro converter-specific doc property, and verifies that the former is used for the field's description?